### PR TITLE
Adds new chemical: spray tan

### PR DESCRIPTION
**Pull request by lordpidey**
Adds a new chemical, spray tan, which darkens the user's skin/scales.  Metabolizes extremely fast EXCEPT when overdosing.

### CODE REVIEW

**Comment by RemieRichards** [comment]
The overdose messages are a bit much.

**Comment by lordpidey** [comment]
Do you mean they come out too often, or that the sayings themselves are excessive?

**Comment by RemieRichards** [comment]
Well, Both actually. Prob(20) and Prob(33) are pretty high for a processing proc to display a message.
And the messages are... yeah a bit too wacky.

**Comment by lordpidey** [comment]
Aright, I've reduced it to 10 and 7.  I've also reduced the ridiculousness of some messages slightly.

**Comment by RemieRichards** [comment]
Yeah, that works.

File: code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm [hunk 1]
```diff
@@ -253,6 +253,92 @@
...
+			N.skin_tone = "orange"
+			N.hair_style = "Spiky"
+			N.hair_color = "000"
+			N.update_hair()
+		if(MUTCOLORS in N.dna.species.specflags) //Aliens with custom colors simply get turned orange
+			N.dna.features["mcolor"] = "f80" 
+			N.regenerate_icons()
+		N.update_body()
+	if(prob(10))
+		M.say(pick("Check these sweet biceps bro!", "Deal with it.", "CHUG! CHUG! CHUG! CHUG!", "Winning!", "NERDS!"))
```
Comment (freerealestate) [inline code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm]: You need something like "My name is John and I hate every single one of you," or basically any /fit/ meme.

**Comment by kevinz000** [comment]
Should add a way to revert the effects

File: code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm (hunk 1 above)
Comment (Miauw62) [inline code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm]: Do you even lift?

**Comment by vista-** [comment]
@kevinz000 Once a douchebag, always a douchebag.

File: code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm [hunk 2]
```diff
@@ -253,6 +253,92 @@
...
+	if(prob(7))
+		M.visible_message(pick("<b>[M]</b>'s collar pops up without warning.</span>", "<b>[M]</b> flexes their arms."))
```
Comment (GunHog) [inline code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm]: Nonsensical. What if the mob is naked, wearing a space suit, or any other situation where he would not have a collar?

**Comment by daman997** [comment]
for what purpose :+1:

**Comment by Aranclanos** [comment]
remove all the memes

File: code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm (hunk 2 above)
Comment (Supermichael777) [inline code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm]: Then its his collar BONE Bro. His bones.

**Comment by Supermichael777** [comment]
can it be applied via spray bottle?

File: code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm (hunk 2 above)
Comment (Miauw62) [inline code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm]: for liggers it can be their snowflakey collar frill thingies

**Comment by freerealestate** [comment]
well this is a thing now

**Pull request merged**